hera_sim/rfi.py

```diff
@@ -69,36 +86,8 @@ def gen_rfi(self, fqs, lsts, rfi=None):
         return rfi
 
 
-HERA_RFI_STATIONS = [
-    # FM Stations
-    (0.1007, 1.0, 1000 * 100.0, 10.0, 100.0),
-    (0.1016, 1.0, 1000 * 100.0, 10.0, 100.0),
-    (0.1024, 1.0, 1000 * 100.0, 10.0, 100.0),
-    (0.1028, 1.0, 1000 * 3.0, 1.0, 100.0),
-    (0.1043, 1.0, 1000 * 100.0, 10.0, 100.0),
-    (0.1050, 1.0, 1000 * 10.0, 3.0, 100.0),
-    (0.1052, 1.0, 1000 * 100.0, 10.0, 100.0),
-    (0.1061, 1.0, 1000 * 100.0, 10.0, 100.0),
-    (0.1064, 1.0, 1000 * 10.0, 3.0, 100.0),
-
-    # Orbcomm
-    (0.1371, 0.2, 1000 * 100.0, 3.0, 600.0),
-    (0.1372, 0.2, 1000 * 100.0, 3.0, 600.0),
-    (0.1373, 0.2, 1000 * 100.0, 3.0, 600.0),
-    (0.1374, 0.2, 1000 * 100.0, 3.0, 600.0),
-    (0.1375, 0.2, 1000 * 100.0, 3.0, 600.0),
-    # Other
-    (0.1831, 1.0, 1000 * 100.0, 30.0, 1000),
-    (0.1891, 1.0, 1000 * 2.0, 1.0, 1000),
-    (0.1911, 1.0, 1000 * 100.0, 30.0, 1000),
-    (0.1972, 1.0, 1000 * 100.0, 30.0, 1000),
-    # DC Offset from ADCs
-    # (.2000, 1., 100., 0., 10000),
-]
-
-
 # XXX reverse lsts and fqs?
-def rfi_stations(fqs, lsts, stations=HERA_RFI_STATIONS, rfi=None):
+def rfi_stations(fqs, lsts, stations=_get_hera_stations(), rfi=None):
```

Comment:
Again, use None, and call the func inside the func body. Though, I would have thought that applying `@_defaults` should have dealt with all of this.